packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -66,13 +64,16 @@ export const firmwareUpdate = () => async (dispatch: Dispatch, getState: GetStat
 
     dispatch(setStatus('downloading'));
 
+    // update to same variant as is currently installed
+    const toBtcOnly = !!isBitcoinOnly(device);
```

Comment:
No need to cast this into a boolean as it already returns a boolean.

Reply:
actually not true, it returns boolean | undefined, and I need it to be boolean later in the code.  But I am going to change the function to return boolean, that will be cleaner.